apps/site/assets/ts/models/__tests__/alert-test.ts

```diff
@@ -368,3 +369,18 @@ describe("alertsAffectingBothDirections", () => {
     expect(result[2].id).toBe("5555");
   });
 });
+
+describe("hasFacilityAlert", () => {
+  test("should return true when facilityId matches facility in alert else false", () => {
+    let facilityId = "ele-125";
+    const facilityAlert = {
+      ...alert1,
+      informed_entity: {
+        facility: ["ele-125"]
+      } as InformedEntitySet
+    };
+
+    expect(hasFacilityAlert(facilityId, [facilityAlert])).toBeTruthy();
+    expect(hasFacilityAlert(facilityId, [alert2])).toBeFalsy();
```

Comment:
nit-pick: I think we should error on using `toBe(true)` and `toBe(false)` as Kayla pointed out the type corercion is affected by this.  Meaning `expect(undefined).toBeFalsy()` returns true.  I don't think this test is directly affected by it, but it's a good habit to get into.